INSTALL.rst

```diff
@@ -42,9 +69,7 @@ Disable SELinux
 The setup described below runs into problems with SELinux related to the sqlite
 database. For now the recommended solution is to simply disable SELinux. In
 future releases, we will support and recommend the use of SELinux with another
-DBMS, such as MySQL.
-
-::
+DBMS, such as MySQL.::
```

Comment:
I would take out the details about exactly what's causing the error, and just say it doesn't work with SELinux, and refer to bug #456